pandora-server-directory/src/database/databaseStructure.ts

```diff
@@ -66,6 +67,7 @@ export const DatabaseAccountSchema = z.object({
 	profileDescription: z.string().default('').transform(ZodTruncate(LIMIT_ACCOUNT_PROFILE_LENGTH)),
 	characters: ZodCast<ICharacterSelfInfoDb>().array(),
 	settings: DirectoryAccountSettingsSchema.catch(() => cloneDeep(ACCOUNT_SETTINGS_DEFAULT)),
+	settingsCooldowns: ZodCast<DirectoryAccountSettingsCooldowns>().catch(() => ({})),
```

Comment:
This needs to be a schema for the migration to actually happen - `ZodCast` simply ignores the value and passes it along as-is - same as using `as` to cast the type - so the `catch` will never trigger.
```suggestion
	settingsCooldowns: DirectoryAccountSettingsCooldownsSchema.catch(() => ({})),
```